src/styles/typography/index.md.njk

```diff
@@ -13,9 +13,14 @@ show_page_nav: true
 
 If your service is on the service.gov.uk subdomain you must use the GDS Transport font.
 
+### When not to use the GDS Transport font
+
 You should use an alternative typeface like Helvetica or Arial for services that are publicly available on different domains.
```

Comment:
Since we're breaking up this section to insert a new heading, I wonder whether it's now less clear what we refer to when we say 'different domains'. 

Should we say 'for services that are publicly available on domains other than service.gov.uk'

Reply:
That's a good shout. Would this work?

```suggestion
For services that are publicly available on domains other than GOV.UK, you should use an alternative typeface like Helvetica or Arial.
```

Reply:
@EoinShaughnessy Sounds good, except it should be 'domains other than service.gov.uk'.

Reply:
Could the wording suggested in the next comment down work?

> If your service is publicly available but on a different domain, use an alternative typeface like Helvetica or Arial.

Or should we definitely mention 'service.gov.uk'?